app/models/pom_task.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class PomTask
+  attr_reader :offender_name, :offender_first_name, :prison_id, :offender_number, :type, :parole_review_id
+
+  # Full name and first name passed individually in case of situations where first name isn't easily determinable from full name.
+  def initialize(offender_full_name, offender_first_name, prison_id, offender_no, type, parole_review_id = nil)
```

Comment:
Is it worth passing in these values individually, could we pass in the offender?